hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java

```diff
@@ -165,14 +171,43 @@ public boolean rename(Path src, Path dst) throws IOException {
     return true;
   }
 
+  private boolean renameV1(OFSPath srcPath, OFSPath dstPath) {
+    OzoneManagerProtocolProtos.OMRequest omRequest =
+        getRenameKeyRequest(srcPath, dstPath);
+    try {
+      submitRequest(omRequest);
```

Comment:
Please add a null check for omRequest. Otw, in error case NPE will get thrown and good to avoid NPE in logs.

can do something like:
```
if(omRequest != null) { 
    submitRequest(omRequest);.
    return true;
}
return false;
```

Reply:
Done.